build/BuildSteps.cs

```diff
@@ -648,6 +648,18 @@ public static void DotnetPublishForNupkg()
                                 $"-c Release -f net8.0");
         }
 
+        public static void DotnetPublishForCustomHost()
+        {
+            // This publishes to the /bin/Release/$targetFramework$/publish
+            Shell.Run("dotnet", $"publish {Settings.CustomHostProjectFile} " +
+                                $"/p:BuildNumber=\"{Settings.BuildNumber}\" " +
```

Comment:
I do not think BuildNumber, NoWorkers, IntegrationBuildNumber, CommitHash are used anywhere when building native host.